axelrod/tests/unit/test_resultset.py

```diff
@@ -154,6 +155,24 @@ def test_payoff_matrix(self):
         self.assertEqual(self.round_matrix(
             rs.payoff_stddevs, 2), self.expected_stddevs)
 
+    def test_wins(self):
+        rs = axelrod.ResultSet(self.players, 5, 2, self.test_outcome)
+        wins = rs.wins
+        self.assertEqual(wins, self.expected_wins)
+
+    def test_winner(self):
```

Comment:
Would a docstring here be helpful? I'm just glancing and not immediately sure what's happening.